src/service.ts

```diff
@@ -219,4 +217,15 @@ export default class RabbitMQService {
       this.assertedQueues.add(queueName)
     }
   }
+
+  private _registerQueue(queueName: string, cb: (r: any) => Promise<void>): void {
```

Comment:
Would it make sense to change the name to `registerSingleQueue` so the difference with `registerQueue` is more explicit?